package/boot/uboot-kirkwood/patches/100-dns325.patch

```diff
@@ -0,0 +1,85 @@
+--- a/configs/dns325_defconfig
++++ b/configs/dns325_defconfig
+@@ -6,7 +6,7 @@ CONFIG_SYS_TEXT_BASE=0x600000
+ CONFIG_TARGET_DNS325=y
+ CONFIG_ENV_SIZE=0x20000
+ CONFIG_ENV_OFFSET=0xE0000
+-CONFIG_NR_DRAM_BANKS=2
++CONFIG_NR_DRAM_BANKS=1
```

Comment:
https://github.com/u-boot/u-boot/commit/e7e650e8bdb09ac46da017250394f8c589df10bb 
According this, `CONFIG_NR_DRAM_BANKS=1` was configured intentionally.

Reply:
Moved to separate patch 011-arm-kirkwood-dns325-Correct-CONFIG_NR_DRAM_BANKS-parameter.patch as a backport from a newer uboot version (v2022.04-rc2)

Without this patch, the boot process freeze on DRAM initialization:
U-Boot 2020.04 (Feb 22 2022 - 00:42:39 +0000)
D-Link DNS-325

SoC:   Kirkwood 88F6281_A1
DRAM:  
<freeze>